config/api.yml

```diff
@@ -1499,14 +1499,14 @@
     :collection_actions:
       :get:
       - :name: read
-        :identifier: miq_event
+        :identifier: miq_event_definition
       :post:
       - :name: query
-        :identifier: miq_event
+        :identifier: miq_event_definition
     :resource_actions:
       :get:
       - :name: read
-        :identifier: miq_event
+        :identifier: miq_event_definition
```

Comment:
Here, it might make sense to use `miq_event_show_list` instead of the current one.

Reply:
I believe since both `miq_event_show` and `miq_event_show_list` are children of `miq_event_definition`, changing to either of the children wouldn't require a migration to update as those with the current permission wouldn't loose anything, and it would only allow for more granular permissions later.

However, correct me if I am wrong on this.

Reply:
@NickLaMuro it makes sense to go with granular permissions.

Reply:
Roger.  Done.